universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -199,6 +237,14 @@ private ContainerTag blockEditPanel(
               .withClasses(Styles.MX_4, Styles.MY_1, Styles.INLINE));
     }
 
+    if (blockDefinition.isRepeater()) {
```

Comment:
Nit (non-blocking): since we only need isRepeater() here, could we just pass in the boolean?

Reply:
done